packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -475,7 +473,7 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   const onRenderChatContent = useCallback(
     (): JSX.Element => (
       <ChatComposite
-        {...chatProps}
+        adapter={chatAdapter}
```

Comment:
Is this really what we want? Are there any other props that we might need from chatProps?

Reply:
There was only adapter before but it was created as a separate object